OxfordBodleian/Juel-Jensen/BDLaethe39.xml

```diff
@@ -62,11 +62,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#12r"/>.</note>
                                 </extent> 
                                 <condition key="deficient">
+                                    <!-- What follows is the condition of the entire MS -->
                                     The manuscript is in bad condition. Tears, losses, and insect droppings are present, especially along the external margins.
-                                  Most folia are unevenly distorted, stained, and darkened along the margins.
-                                  The sewing is broken throughout. Most bifolia became separated and the single folia are stitched
-                                  to strips of parchment or textile placed around the quires.
-                                  </condition>
+                                    Most folia are unevenly distorted, stained, and darkened along the margins.
```

Comment:
What about: "The two codicological units forming the manuscript seem to share the same conservation history. Most folia are unevenly distorted, stained, and darkened along the margins, and present tears and lacunas. The sewing is broken throughout. Most bifolia became separated and the single folia are stitched to strips of parchment or textile placed around the quires.  A protective strip of textile is sewn around `<ref` target="#q5 #q7 #q9 #q11 #q12 #q13 #q14 #q17 #q18"/>. A parchment support has been added around `<ref` target="#q1"/>.  `<ref` target="#q19 originates from another codex and is currently held together very precariously by means of one running stitch. One or more leaves are missing after `<locus` target="#88"/>, with loss of text.  All leaves are stained with dirt and some portions of text are illegible due to faded ink."

Reply:
Dear Eliana, everything seems fine to me: the text is fine as it is. Do you want to insert it yourself, delete the condition elements of p1 and p2 and open a PR? It's also fine with just me as a reviewer, I'll merge it.